content.js

```diff
@@ -98,22 +102,21 @@ chrome.runtime.onMessage.addListener( // Detect URL Changes
 });
 
 //check for hotkey pressed
-document.onkeydown = function(e){
-  e = e || window.event;
-  var key = e.which || e.keyCode;
-
-  let video = document.getElementById("movie_player");
-
-  //is the video in focus, otherwise they could be typing a comment
-  if (document.activeElement === video) {
-    if(key == 186){
-      //semicolon
-      startSponsorClicked();
-    } else if (key == 222) {
-      //single quote
-      submitSponsorTimes();
+document.onkeydown = function(e) {
+    e = e || window.event;
+    let video = document.getElementById("movie_player");
+    //is the video in focus, otherwise they could be typing a comment
+    if (document.activeElement === video) {
+        if (e.code == hotkeys.submit) {
+            if (hasData) { // Submit!!!
+                submitSponsorTimes();
+                hasData = false;
+            } else { // First time
+                startSponsorClicked();
+                hasData = true;
+            }
+        }
```

Comment:
This won't work, as clicking once indicates the start of a sponsor, then clicking again should indicate the end. I think a separate button (I used single quote) should be used for the submitting hot key. Someone might want to submit multiple at once.

Reply:
Code works however needs change